src/views/AOncoPrint.ts

```diff
@@ -395,7 +395,7 @@ export abstract class AOncoPrint extends AView {
     $cells
       .attr('data-title', (d) => d.name) //JSON.stringify(d))
       .attr('data-id', (d) => d.sampleId)
-      .attr('data-cnv', (d) => String(d.cn))
+      .attr('data-cnv', (d) => String(isMissingCNV(d.cn) ? Categories.unknownCopyNumberValue : d.cn))
```

Comment:
This was missing from https://github.com/Caleydo/tdp_gene/pull/132 but is necessary.